README.md

```diff
@@ -1 +1,246 @@
-# microverse-personal-portfolio
\ No newline at end of file
+
+
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
```

Comment:
- [ ] For clean code and as it is mentioned in this README template, kindly ensure that you remove all the comments and instructions after you have finished to use it.